### ISSUE TITLE
R50 train parameter

### ISSUE THREAD

**Issue by dianxin556** [biubug6/Pytorch_Retinaface#67]
Is the R50 result displayed on the page consistent with the training parameters of the mxnet version? I see that the training scale is 800 * 800 in R50‘s  config.

**Comment by zehuichen123** [comment]
Same question. And the initial learning rate in the paper is 1e-2, while 1e-3 in this repo here. Have you checked if we can enlarge the lr while reducing the training epochs?

**Comment by zehuichen123** [comment]
@dianxin556  have you ever tried one 640x640 trained model? My device is not allowed to train 24 images for 840x840 :(

**Comment by zehuichen123** [comment]
Hi, I've trained one 640x640 baseline, the result is
==================== Results ====================
Easy   Val AP: 0.9270121584603086
Medium Val AP: 0.9234432537094537
Hard   Val AP: 0.884572269484029

**Comment by praneet195** [comment]
can you share your training config parameters?
